docs/relnotes.md

```diff
@@ -7,18 +7,66 @@ hide:
 
 This is the list of changes to Taipy releases as they were published.
 
-## Community edition: 1.2 (In progress)
+## Community edition: 2.0
+
+Published on TO_BE_DEFINED.
+
+[`taipy` 2.0](https://pypi.org/project/taipy/2.0.0/) contains the latest
+[`taipy-config` 2.0](https://pypi.org/project/taipy-config/2.0.0/),
+[`taipy-gui` 2.0](https://pypi.org/project/taipy-gui/2.0.0/),
+[`taipy-core` 2.0](https://pypi.org/project/taipy-core/2.0.0/) and
+[`taipy-rest` 2.0](https://pypi.org/project/taipy-rest/2.0.0/) packages.
+
+#### Features
+
+**`taipy-core`** 2.0.0
+
+   - New data node: SQLTableDataNode. It represents a table in a SQL database.
```

Comment:
"New data node: SQLTableDataNode. It represents a table in a SQL database."
->
"New data node class: `SQLTableDataNode`. It represents a table in an SQL database.`

Same remark for JSON (not the 'an' thing of course)